### PR TITLE
Shiny Prior Elicitation tool

### PR DESCRIPTION
**Pull request by mdietze**
## Description

First pass at a Shiny tool to help modeling teams generate priors for their models. Follows the general principle of eliciting quantiles from the extremes inward. Then fits a series of candidate distributions and asks user to select one.

Not yet coupled to BETY (can't grab info or insert), but that's the end goal. For now spits out an RData object from which prior choice and parameters can be culled.

Important caveat: I suck at Shiny. I'm sure there's a much more elegant way to do this and will be soliciting help from others for further improvements
## Motivation and Context

Creating priors is a major conceptual bottleneck for modeling teams -- most haven't thought about this rigorously and are prone to throw uninformative uniform distributions on everything
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) 
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
## Checklist:

- [X] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

 

@tonygardella who's on review for me this week?

### CODE REVIEW

File: shiny/Elicitation/DESCRIPTION [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Type: Shiny
+Title: Expert Prior Elicitation
+License: FreeBSD + file LICENSE
+Author: Mike Dietze <dietze@bu.edu> 
+Author: Rob Kooper <kooper@illinois.edu>
+AuthorUrl: http://www.ncsa.illinois.edu/assets/php/directory/contact.php?contact=kooper
+Tags: PEcAn
+DisplayMode: Normal
```
Comment (robkooper) [inline shiny/Elicitation/DESCRIPTION]: Add Depends: SHELF

File: shiny/Elicitation/README.md [hunk 2]
```diff
@@ -0,0 +1 @@
+
```
Comment (robkooper) [inline shiny/Elicitation/README.md]: Maybe add some text here how it works?

File: shiny/Elicitation/DESCRIPTION (hunk 1 above)
Comment (mdietze) [inline shiny/Elicitation/DESCRIPTION]: done

File: shiny/Elicitation/README.md (hunk 2 above)
Comment (mdietze) [inline shiny/Elicitation/README.md]: done

**Pull request merged**